packages/shared/src/modules/section-statuses/libs/validation-schemas/validation-schemas.ts

```diff
@@ -0,0 +1,4 @@
+export { sectionStatusCreateBody } from "./section-status-create-body.validation-schema.js";
+export { sectionStatusUpdateBody } from "./section-status-update-body.validation-schema.js";
+export { sectionStatusUpdateQuery } from "./section-status-update-query.validation-schema.js";
+export { sectionStatusesGetAllQuery } from "./section-statuses-get-all-query.validation-schema.js";
```

Comment:
```suggestion
export { sectionStatusesGetAllQuery } from "./section-status-get-all-query.validation-schema.js";
```
Don't forget about the validation object variable name